.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@
 #
 
 default_language_version:
-    python: python3.8
```

Comment:
Made this change because I was developing in an environment where python3.8 was not available. Any problem with having this generic for python3?